File: tools/calibrate-pipeline/fixtures/ast-diff-trivial-check/bad-05-py-rename-only/output.py
import time

import pytest
from selenium import webdriver
from selenium.webdriver.common.by import By


@pytest.fixture
def browser():
    drv = webdriver.Chrome()
    drv.implicitly_wait(5)
    yield drv
    drv.quit()


def test_admin_authenticates(browser):
    browser.get("https://app.acme.test/signin")
    browser.find_element(By.ID, "userField").send_keys("admin@acme.test")
    browser.find_element(By.ID, "pwField").send_keys("supersecret")
    browser.find_element(By.XPATH, "//form//button[@type='submit']").click()

    time.sleep(2)

    banner = browser.find_element(By.CSS_SELECTOR, ".home-banner").text
    assert banner == "Welcome, Administrator"

File: tools/calibrate-pipeline/fixtures/ast-diff-trivial-check/bad-04-java-rename-only/output.java
package com.acme.shop.tests;

import org.junit.jupiter.api.AfterEach;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

import static org.junit.jupiter.api.Assertions.assertTrue;

public class AuthTest {

    private WebDriver browser;

    @BeforeEach
    void initialize() {
        browser = new ChromeDriver();
        browser.get("https://app.acme.test/signin");
    }

    @AfterEach
    void cleanup() {
        browser.quit();
    }

    @Test
    void adminAuthLandsOnHome() throws InterruptedException {
        browser.findElement(By.id("usernameField")).sendKeys("admin@acme.test");
        browser.findElement(By.id("pwField")).sendKeys("supersecret");
        browser.findElement(By.cssSelector("button.primary")).click();
        Thread.sleep(2000);
        WebElement homeBanner = browser.findElement(By.cssSelector(".home-banner"));
        assertTrue(homeBanner.isDisplayed());
    }
}
